BioscoopSysteemAPI/BioscoopSysteemAPI/Models/Movie.cs

```diff
@@ -19,12 +20,10 @@ public class Movie
         public byte AllowedAge { get; set; }
 
         [MaxLength(255)]
-        public string? ImageUrl { get; set; }
-        
+        public string? ImageUrl { get; set; }       
         public string Language { get; set; }
         public bool Subtitles { get; set; }
         public string Genre { get; set; }
         public string Specials { get; set; }
```

Comment:
Deze zijn overgenomen vanuit dev met het resolven van de mergeconflicten. Het enigste verschil is dat ze met een hoofdletter beginnen.

Geen idee of dit ergens in je code nog aangepast moet worden?

Dit geldt voor het Movie model en de Movie DTO's

Reply:
Alleen in de movieservice, maar heb ze aangepast ik ga mergen